index.html

```diff
@@ -65,12 +80,37 @@
 	</div>
 </section>
 
+<div class="hide">
+	<nav class="flex-menu-bar2">
+	<div class="brand">
+		Mamayoun Ali
+	</div> 
```

Comment:
- [ ] Brand position is broken in the mobile version please fix it.

![image](https://user-images.githubusercontent.com/38635158/173106550-8938c667-53b9-4e78-9a46-f65d3837b444.png)

- [ ] It has some space left and on top of the page please fix it.

![image](https://user-images.githubusercontent.com/38635158/173107852-793eb082-1cdd-4f50-a5c8-36ae5d173da3.png)